src/combobox/ComboBox.ts

```diff
@@ -169,19 +162,19 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 	private _onInputKeyDown(event: KeyboardEvent) {
 		const { results = [] } = this.properties;
 
-		switch (event.keyCode) {
-			case keys.up:
+		switch (event.which) {
```

Comment:
Sounds like `which`, although being the most poorly named, is the best option to go with.

Regardless, we do still want a `keys` module, right? Even if we're able to go with `event.key` it will be nice to be able to do comparisons against `keys.ArrowDown` etc.